File: Dante1/Module1.02/Zadanie2.11/main.c
#include <stdio.h>

int main()
{
    int number;
    printf("Enter number: ");
    int check = scanf("%i",&number);
    if(check != 1)
    {
        printf("Incorrect input");
        return 1;
    }
    else if(number < 0 || number > 10)
    {
        printf("Incorrect input data");
        return 2;
    }
    else if(number == 0)
    {
        printf("zero");
    }
    else if(number == 1)
    {
        printf("jeden");
    }
    else if(number == 2)
    {
        printf("dwa");
    }
    else if(number == 3)
    {
        printf("trzy");
    }
    else if(number == 4)
    {
        printf("cztery");
    }
    else if(number == 5)
    {
        printf("piec");
    }
    else if(number == 6)
    {
        printf("szesc");
    }
    else if(number == 7)
    {
        printf("siedem");
    }
    else if(number == 8)
    {
        printf("osiem");
    }
    else if(number == 9)
    {
        printf("dziewiec");
    }
    else if(number == 10)
    {
        printf("dziesiec");
    }
    return 0;
}

File: Dante1/Module1.02/Zadanie2.11/main.rs
fn main() {
    let mut input: String = String::new();
    std::io::stdin().read_line(&mut input).expect("Error");
    let number: i32 = input.trim().parse().expect("Error");

    if(number < 0 || number > 10)
    {
        println!("Incorrect input data");
    }
    else if(number == 0)
    {
        println!("zero");
    }
    else if(number == 1)
    {
        println!("jeden");
    }
    else if(number == 2)
    {
        println!("dwa");
    }
    else if(number == 3)
    {
        println!("trzy");
    }
    else if(number == 4)
    {
        println!("cztery");
    }
    else if(number == 5)
    {
        println!("piec");
    }
    else if(number == 6)
    {
        println!("szesc");
    }
    else if(number == 7)
    {
        println!("siedem");
    }
    else if(number == 8)
    {
        println!("osiem");
    }
    else if(number == 9)
    {
        println!("dziewiec");
    }
    else if(number == 10)
    {
        println!("dziesiec");
    }
}
